csharp/src/Ice/ProxyComparer.cs

```diff
@@ -0,0 +1,64 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+using System.Collections.Generic;
+
+namespace ZeroC.Ice
+{
+    /// <summary>Represents a <see cref="IObjectPrx">object proxy</see>comparison operation that uses specific rules.
```

Comment:
What about:
Represents an object proxy comparison operation based on all or only some of the proxy properties. The Default property delegates to the implementation of IEquatable<T> provided by IObjectPrx.

(and no need to describe the other fields in this summary).

Reply:
fixed